consoleApp/src/main/java/com/infoshare/database/FileDb.java

```diff
@@ -0,0 +1,112 @@
+package com.infoshare.database;
+
+import com.infoshare.domain.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileDb implements DB {
+    private static final String VOLUNTEER_DB_FILE_NAME = "Volunteer.csv";
+    private static final String REQUEST_DB_FILE = "NeedRequest.csv";
+    private static final SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss"); // uzywamy df do formatowania i parsowania dat
+
+    public FileDb() {
+    }
+
```

Comment:
domyślny konstruktor sam się robi, jeżeli nie ma innego konstruktora to chyba ten nie będzie potrzebny